docs/source/dev_guide/pre_release_testing.rst

```diff
@@ -0,0 +1,26 @@
+***************************************
+Testing directions for making a release
+***************************************
+
+1. Have three shells open: one on Chrysalis, one on Compy, and one on Perlmutter. Do the following steps on each machine.
+
+2. ``cd`` to the ``zppy`` directory.
+   
+3. Run ``git log``. The most recent commit should match the most recent commit on https://github.com/E3SM-Project/zppy/commits/main. This ensures that testing will use the latest code changes.
+
+4. Set up a new development environment. This ensures that testing will use the latest conda changes. Note that you will need to run ``conda remove -n zppy_dev_pre_rc<#> --all`` first if you have previously done this step. ::
+
+     mamba clean --all
+     mamba env create -f conda/dev.yml -n zppy_dev_pre_rc<#>
+     conda activate zppy_dev_pre_rc<#>
+     pip install .
+
```

Comment:
There needs to be a step added here to ensure the correct environments are being used!!!

In `tests/integration/utils.py`, the last line `generate_cfgs()` needs to be changed to `generate_cfgs(unified_testing=True)`. 

If this is not done, then `environment_commands=""` by default, which will trigger the following in https://github.com/E3SM-Project/zppy/blob/main/zppy/__main__.py#L170:
```
    if config["default"]["environment_commands"] == "":
        machine = config["default"]["machine"]
        config["default"][
            "environment_commands"
        ] = f"source {unified_base}/load_latest_e3sm_unified_{machine}.sh"
```
That is, the latest RELEASED version of Unified will be used for all the packages.

Additionally, `environment_commands_test` needs to be updated for Chrysalis, Compy, and Perlmutter. Otherwise, the test will try to use a Unified RC launch script that probably doesn't even exist anymore. This parameter should be updated to be some form of a Unified launch script, since Unified is the only environment that all the `zppy` tasks can run from. (For instance, `ncclimo` isn't in the `zppy` dev environment).

Furthermore, `diags_environment_commands` should be updated for Chrysalis, Compy, and Perlmutter. Otherwise,  the `atm_monthly_180x360_aave_environment_commands` subtask will use an older dev environment for E3SM Diags than we want.

The updated parameters can be merged to keep the file up to date, but the `generate_cfgs(unified_testing=True)` change should not be merged, since in most cases we are not testing for Unified.

Reply:
We'll need directions for setting `diags_environment_commands` too:
```
# `cd` to e3sm_diags directory
$ git fetch upstream main
$ git rebase upstream/main
$ mamba clean --all
$ mamba env create -f conda/dev.yml -n e3sm_diags_<date>
$ conda activate e3sm_diags_<date>
$ pip install .
# Now set `diags_environment_commands` to `e3sm_diags_<date>`
```